src/language-js/printer-estree.js

```diff
@@ -2707,7 +2711,10 @@ function printPathNoParens(path, options, print, args) {
               concat([
                 line,
                 "extends ",
-                indent(join(concat([",", line]), path.map(print, "extends")))
+                (extendsDoc =>
+                  n.extends.length === 1 ? extendsDoc : indent(extendsDoc))(
+                  join(concat([",", line]), path.map(print, "extends"))
+                )
```

Comment:
How do you feel about creating a helper function for this?

```js
const indentIf = (condition, doc) => condition ? indent(doc) : doc;
```

This would eliminate the need for an IIFE and make the code a little clearer IMO.

Reply:
A general `identity` would be better I guess:

```js
(condition ? identity : indent)(doc)

function identity(x) {
  return x;
}
```